browser/brave_content_browser_client.cc

```diff
@@ -203,6 +208,15 @@ bool BraveContentBrowserClient::HandleExternalProtocol(
   }
 #endif
 
+#if BUILDFLAG(GEMINI_ENABLED)
+  if (gemini::IsGeminiProtocol(url)) {
+    gemini::HandleGeminiProtocol(url, std::move(web_contents_getter),
+                                   page_transition, has_user_gesture,
+                                   initiating_origin);
```

Comment:
nit, unindent to after the opening param w/ `url`  (back 2 spaces)

Reply:
fixed